alot/settings/manager.py

```diff
@@ -68,15 +68,11 @@ def reload(self):
 
     def read_notmuch_config(self):
         """parse notmuch's config file from path"""
-        if self.notmuch_rc_path is not None:
```

Comment:
Why is this safe? I mean, what if `self.notmuch_rc_path` is `None` because it is unset due to a missing alot config?
Then two lines below should raise an `ConfigError` which we don't catch here?